packages/swingset-liveslots/src/collectionManager.js

```diff
@@ -348,6 +348,12 @@ export function makeCollectionManager(
       updateEntryCount(1);
     }
 
+    function addToSet(key) {
+      if (!has(key)) {
+        init(key, null);
```

Comment:
`init` starts with its own `has()` call, so duplicate adds will cause two DB reads before doing the write.

I don't think that's worth optimizing now (it would require adding a `alreadyChecked` option to `init`, or splitting `init` into `checkedInit` and `uncheckedInit`, or something), but if duplicate Set additions turn out to be a common thing, we could improve it in the future.

Maybe add a one-line comment, `// TODO: optimize out the duplicate has() call`, so we'll know where to look.

Reply:
Yeah, this bugged me too, but other code paths already get stuck with extraneous `has` calls so I figured this actually wasn't as big a change as one might think.  It's one of the reasons I dislike the `init` vs. `set` convention.

I will add a suitable comment and then land the thing.

Reply:
Since I was already in there fixing MarkM's style whinge, I went ahead and fixed this too.